README.md

```diff
@@ -228,7 +228,7 @@ List of Interface Layers
 | keystone-admin | [Repo](https://github.com/openstack/charm-interface-keystone-admin) | [Docs](https://github.com/openstack/charm-interface-keystone-admin#readme) | keystone-admin | keystone-admin:identity-admin interface to use shared admin API credentials |
 | keystone-credentials | [Repo](https://github.com/openstack/charm-interface-keystone-credentials) | [Docs](https://github.com/openstack/charm-interface-keystone-credentials#readme) | keystone-credentials | keystone-credentials: Interface for integrating with Keystone identity credentials |
 | keystone-domain-backend | [Repo](https://github.com/openstack/charm-interface-keystone-domain-backend) | [Docs](https://github.com/openstack/charm-interface-keystone-domain-backend#readme) | keystone-domain-backend | Interface for integration of subordinate charms providing domain specific identity backends |
-| keystone-fid-service-provider | [Repo](https://github.com/openstack-charmers/charm-interface-keystone-fid-service-provider) | [Docs](https://github.com/openstack-charmers/charm-interface-keystone-fid-service-provider#readme) | keystone-fid-service-provider | An interface to connect a federated identity provider to the Keystone charm |
+| keystone-fid-service-provider | [Repo](https://github.com/openstack/charm-interface-keystone-fid-service-provider) | [Docs](https://github.com/openstack/charm-interface-keystone-fid-service-provider#readme) | keystone-fid-service-provider | An interface to connect a federated identity provider to the Keystone charm |
```

Comment:
Should this be https://opendev.org/openstack/charm-interface-keystone-fid-service-provider ?

Reply:
Also are we getting rid of the openstack-charmers repos? So many repos!

Reply:
Corey,

These are the pointers to the repositories to pull from so github.com links are correct.

Yes, we will remove the openstack-charmers repos for these as soon as this lands.